1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,15 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let num= 0 ;
+  let arr =[]
+  while(n > 0){
+   arr.push(num)
+   num += 2
+   n--
+  }
+  console.log(arr)
```

Comment:
nice! I like this solution. Usually the standard way to look at even/odd number is using %2, but this is really nice too